packages/node_modules/@webex/plugin-rooms/src/rooms.js

```diff
@@ -14,7 +14,7 @@ import {
   deconstructHydraId
 } from '@webex/common';
 
-const debug = require('debug')('memberships');
+const debug = require('debug')('rooms');
```

Comment:
Do we know why we're using `require()` here? Not sure why we are using mixed-import methods, if I'm understanding this correctly.

Reply:
Can't say.  Its a pattern that is consistent across all the mocha tests that I've ever tried to modify.  My only goal here was to fix the fact that the rooms test was using "memberships" string instead of "rooms" as the debug trigger.

Happy to remove this and leave that error in place if it gets the PR approved.  This was just a side bit of cleanup I was trying to do ;-)

Reply:
Looks good, this is how this is currently implemented throughout the SDK in tests. We will follow this up internally to determine if this is necessary.